CHANGELOG.md

```diff
@@ -1,24 +1,118 @@
+# v0.5.0
+
+## Enhancements
+
+* Add API for fetching Fulcio configuration (https://github.com/sigstore/fulcio/pull/608)
+* Final challenge result removal 🎉  (https://github.com/sigstore/fulcio/pull/626)
```

Comment:
Can we group the principal refactors into one line with multiple PRs? It'd be Final challenge result removal, Move email principal to package, Move kubernetes principal to package, Move spiffe principal to package, Move github principal to its own package, Use principal in CA abstraction, Refactor x509 extension embedding logic, Add new Issuer and Principal abstractions